chart/Chart.yaml

```diff
@@ -27,7 +27,6 @@ appVersion: "0.0.0"
 dependencies:
   - name: crds
     version: "0.0.0"
```

Comment:
btw do we need to change the publish script to also set the version for the crd chart?

Reply:
This change is already in the PR.